src/prints/upgradePlugin.ts

```diff
@@ -0,0 +1,93 @@
+import { Logger, singleLineString } from 'flex-plugins-utils-logger';
+
+const cracoUpgradeGuideLink = 'https://twilio.com';
```

Comment:
is this mean to just be a placeholder for now?

Reply:
Ya, this is not published yet so I don't have a link for it.